tests/integration/components/frost-bunsen-input-select-test.js

```diff
@@ -128,22 +126,64 @@ describeComponent(...integrationTestContext('frost-bunsen-input-select'), functi
       })
     })
 
-    it('gets async values per query instructions', function (done) {
-      props.model = {
-        modelType: 'resource',
-        labelAttribute: 'label',
-        valueAttribute: 'id',
-        query: {
-          resourceTypeId: '${foo.bar.someOtherProp}',
-          q: 'domainId:${domainId}'
+    describe('when query dependency is met', function () {
+      beforeEach(function (done) {
+        props.model = {
+          modelType: 'resource',
+          labelAttribute: 'label',
+          valueAttribute: 'id',
+          query: {
+            resourceTypeId: '${foo.bar.someOtherProp}',
+            q: 'domainId:${domainId}'
+          }
         }
-      }
-      rootNode = renderWithProps(this, 'frost-bunsen-input-select', props)
-      Ember.run.later(() => {
+        rootNode = renderWithProps(this, 'frost-bunsen-input-select', props)
+        Ember.run.later(function () {
+          done()
+        })
+      })
+
+      it('enables the input', function () {
+        expect(rootNode.find('.frost-select input').prop('disabled')).to.equal(false)
+      })
+
+      it('gets async values per query instructions', function () {
         const expected = ['Resource 2']
         expect($(rootNode).text().indexOf(expected[0]) !== -1).to.eql(true)
         expect($(rootNode).find('li').length).to.equal(expected.length)
-        done()
+      })
+    })
+
+    describe('when query dependency is not met', function () {
+      beforeEach(function () {
+        props.model = {
+          modelType: 'resource',
+          labelAttribute: 'label',
+          valueAttribute: 'id',
+          query: {
+            resourceTypeId: '${foo.bar.someOtherProp}',
+            q: 'domainId:${domainId}'
```

Comment:
Can we add a use-case where the `key` in the `query` object is driven by the `${}` syntax as well? That is, if you agree to update it to support that :)